packages/trader/src/App/Containers/Lazy/lazy.jsx

```diff
@@ -0,0 +1,24 @@
+import PropTypes from 'prop-types';
+import React     from 'react';
+import UILoader  from 'App/Components/Elements/ui-loader.jsx';
+
+const Lazy = (props) => {
```

Comment:
```suggestion
const Lazy = ({ ctor, has_progress, should_load, ...props }) => {
```

Reply:
Added the default property with the proposed changes. Thanks.